test/diff/helper.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2020 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package diff
+
+import (
+	"fmt"
+	"github.com/google/go-cmp/cmp"
```

Comment:
Move this import down to its own section to match import style with other files in pipeline.  e.g.

```go
import (
  "fmt"
  "testing"

  "github.com/google/go-cmp/cmp"
)
```

Reply:
so it has to be alphabetical order?

I already fix this.
but I doubt the change that I made, I'm looking forward to the review

Reply:
> so it has to be alphabetical order?

We split the lists of packages into built in packages provided by go and those that are external libraries.  Thanks for fixing this!